RecoLocalTracker/SiStripClusterizer/BuildFile.xml

```diff
@@ -8,6 +8,9 @@
 <use name="CalibFormats/SiStripObjects"/>
 <use name="CalibTracker/Records"/>
 <use name="RecoLocalTracker/Records"/>
+<use name="CommonTools/UtilAlgos"/>
+<use name="CommonTools/Utils"/>
+<use name="root"/>
```

Comment:
could you please check if all the packages that included in the headers are also in the buildfile? By a quick look, `FWCore/Utilities` seems to be missing (it still compiles, but it makes it harder to trace dependencies).

Reply:
done

Reply:
done